Uplift/Views/Onboarding/SignInView.swift

```diff
@@ -0,0 +1,139 @@
+//
+//  SignInView.swift
+//  Uplift
+//
+//  Created by Belle Hu on 9/14/24.
+//  Copyright © 2024 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct SignInView: View {
+
+    // MARK: - Properties
+
+    @EnvironmentObject var mainViewModel: MainView.ViewModel
+    @StateObject private var loginViewModel = LoginViewModel()
+
+    // MARK: - UI
+
+    var body: some View {
+        VStack {
+            signInHeader
+            login
+            Spacer(minLength: 16)
+            skip
+        }
+    }
+
+    private var skip: some View {
+        Button {
+            mainViewModel.displayMainView = true
+        } label: {
+            Text("Skip")
+                .font(Constants.Fonts.bodyNormal)
+                .foregroundColor(Constants.Colors.gray04)
+        }
+    }
+
+    private var login: some View {
```

Comment:
I would call this "loginButton" to follow your previous conventions, also make it clearer to read